files/js/util.js

```diff
@@ -5,7 +5,8 @@ var Utils = {
         }
     },
     settings: {
-        colours: ["red", "blue", "yellow", "green", "purple", "pink", "turquoise", "brown"]
+        colours: ["red", "blue", "yellow", "green", "brown"]
+        //colours: ["red", "blue", "yellow", "green", "purple", "turquoise", "pink", "brown"]
```

Comment:
The colours are based on a game my mother used to play like 15 years ago actually, I'd prefer to keep them :p